sorts/bubble_sort.py

```diff
@@ -42,6 +42,44 @@ def bubble_sort(collection: list[Any]) -> list[Any]:
     return collection
 
 
+def bubble_sort_recursive(list_data: list, length: int = 0) -> list:
```

Comment:
Please adjust the function signature to be similar...
```suggestion
def bubble_sort_recursive(collection: list[Any]), length: int = 0) -> list[Any]:
```